app/assets/javascripts/admin/addon/components/admin-theme-setting-schema.gjs

```diff
@@ -25,17 +27,16 @@ class Tree {
 @tagName("")
```

Comment:
Could you remove this while here? 🙂 (it's a no-op, that only applies to non-glimmer components)